src/alert.h

```diff
@@ -46,7 +46,7 @@ class CUnsignedAlert
     std::string strStatusBar;
     std::string strReserved;
 
-    ADD_SERIALIZE_METHODS;
+    ADD_SERIALIZE_METHODS
```

Comment:
Its not a style nit because its actually a warning from both my IDE and I'm sure with the right flags, from g++